src/ekam/rules/compile.ekam-rule

```diff
@@ -156,9 +156,15 @@ echo newOutput "${MODULE_NAME}.o.deps"
 read DEPFILE
 
 # Generate the symbol list.
+
+# Additional flags to NM may be provided via NMFLAGS environment variables. We split on | so that
```

Comment:
We don't do this splitting-on-`|` thing with `CXXFLAGS`, `LDFLAGS`, `LIBS`, etc., all of which commonly contain paths. I'd argue there's nothing special about nm that makes it more deserving of such treatment. People just don't put spaces in paths in Unix dev environments.